services/core/config.js

```diff
@@ -149,4 +149,31 @@ config.log.file = process.env.SERVICE_LOG_FILE || 'false';
 config.log.docker_host = process.env.DOCKER_HOST || 'local';
 config.debug = process.env.SERVICE_LOG_LEVEL === 'debug';
 
+config.queue = {
+	defaults: {
+		limiter: {
+			max: 8,
+			duration: 20,
```

Comment:
A comment would be appreciated to indicate the default time units.

```suggestion
			duration: 20, // millisecs
```
or
```suggestion
			duration: 20, // secs
```

Reply:
Done!